release-team/role-handbooks/docs/README.md

```diff
@@ -232,36 +232,53 @@ The middle weeks of the launch are where the Docs Lead and Docs Lead Shadows tra
     This allows us to avoid merge conflicts on release day with `dev-[future release]` and it allows us to easily sunset the current docs under it's new branch `release-[current release]` after we merge `dev-[future release]`.
 
     To merge `master` into `release-[current release]` on your local fork:
-    ```
-    $ git remote add upstream https://github.com/kubernetes/website.git
-    $ git fetch upstream
-    $ git checkout upstream/release-[current release]
-    $ git merge upstream/master
-    $ git checkout -b merged-master-release-[current release]
-    $ git commit -m "merged master into release-[current release] to keep in sync"
-    $ git push origin merged-master-release-[current release]
+    ```bash
+    # Step 0 (if you don't already have a remote called "upstream")
+    git remote add upstream https://github.com/kubernetes/website.git
+    # Step 1
+    git fetch upstream
+    # Step 2
+    git checkout upstream/release-[current release]
+    # Step 3
+    git merge upstream/master
+    # Set the commit message to, eg,
+    # "Merged master into release-[current release] to keep in sync"
```

Comment:
```suggestion
    # "Merge master into release-[current release]"
```

1. Imperative mood for commit messages
2. Keeping things in sync is implied

Reply:
Updated